src/components/Map/index.tsx

```diff
@@ -231,11 +232,13 @@ const Map = () => {
   }, []);
 
   const handleClickForeCastButton = () => {
-    navigate(
-      `${ROUTE.DUST_FORECAST}?sido=${encodeURIComponent(
-        currentSido
-      )}&city=${encodeURIComponent(city)}`
-    );
+    SIDO_NAMES.includes(city)
+      ? navigate(`${ROUTE.RANKING}/${city}`)
+      : navigate(
+          `${ROUTE.DUST_FORECAST}?sido=${encodeURIComponent(
+            currentSido
+          )}&city=${encodeURIComponent(city)}`
+        );
```

Comment:
```suggestion
    SIDO_NAMES.includes(sido)
      ? navigate(`${ROUTE.RANKING}/${sido}`)
      : navigate(
          `${ROUTE.DUST_FORECAST}?sido=${currentSido}&city=${city}`
        );
```
1. 위에 navigate 부분에 `encodeURIComponent`가 없으니까 아래도 제거해서 통일하는 건 어떨까요?!?
2. city랑 sido 네이밍이 혼란스러운데 어떻게 하면 좋을까요?!
3. 현재 작성해주신 코드에서 city와 currentSido는 같은 값인 것 같은데 따로 관리하는 이유가 있을까요!?

Reply:
확인이 조금 늦어질것 같습니다! 다음주 수요일에 확인해보겠습니다.
🙇‍♂️